engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/CustomAction.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import pl.touk.nussknacker.engine.api.process.ProcessName
+
+import java.net.URI
+
+/*
+This is an experimental/wip version.
+FIXME:
+1. There is no validation on BE to check if given action can be processed,
+   eg if an action is some sort of custom process deployment we should validate it against current process status.
+   For now we only disable custom action buttons on FE when action's allowed process states doesn't include current process state.
```

Comment:
Let's add mentions about:
- handling parameters
- additional invocation checks (e.g. is process valid?)

Reply:
:+1: